tools/inference_engine/include/inference_engine.h

```diff
@@ -2,8 +2,8 @@
 #define INFERENCE_ENGINE_H
 
 #include "inference_engine_export.h"
-
-#include <stdint.h>
+#include "inference_engine_error.h"
+#include <cstdint>
```

Comment:
This file is part of C api, `stdint.h` should stay here

Reply:
Visual Assist suggested that change, but you're right that we should keep the C style...